app/(routes)/userPanel/page.tsx

```diff
@@ -0,0 +1,33 @@
+import { UserButton } from "@clerk/nextjs"
+import React from 'react'
+import { Button } from "./Button"
+
+const page = () => {
+  return (
+    <div className="px-[5%] pt-4">
+      <div className="flex justify-end">
+        <UserButton afterSignOutUrl="/" />
+      </div>
+      <main className="text-center pt-4">
+        <h2 className="uppercase font-normal text-2xl">Moje konto</h2>
+        <p>Możesz tu zarządzać swoimi danymi personalnymi jak również przeglądać swoje zamówienia.</p>
+        <div className="container max-w-5xl">
+          <section className="flex justify-center gap-4">
+            <Button>Dodaj pierwszy adres</Button>
```

Comment:
Do you consider localization feature with different laungauges ? (en/pl)
https://phrase.com/blog/posts/nextjs-i18n/
Just to consider :)

Reply:
thanks,
I add this to todo list maybe join this feature later ;-)